ldk/javascript/examples/self-test-loop/src/tests/screen/index.ts

```diff
@@ -109,3 +108,48 @@ export async function performOcr() {
       });
   });
 }
+
+export const testScreenMonitor = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      console.log('Running listenOcrMonitor function...');
+      sleep(1000);
+      const listener = await screen.listenOcrMonitor((ocrEvent) => {
+        sleep(1000);
+        // eslint-disable-next-line @typescript-eslint/no-array-constructor
+        const resultNew = new Array();
+        // eslint-disable-next-line @typescript-eslint/no-array-constructor
+        const resultOld = new Array();
```

Comment:
Is there a reason for using `new Array()` instead of just assigning an empty array `[]`?